src/compose.py

```diff
@@ -9,8 +9,13 @@ def start(settings):
 
 
 def stop(settings):
-    command = "down" if settings["persist_data"] else "down --volumes"
-    run(command, settings)
+    run("stop", settings)
+    if not settings["persist_data"]:
+        p = Popen("docker system prune -a --volumes -f", shell=True)
```

Comment:
I think this should limit itself to the montagu containers, networks and volumes only otherwise it could end up accidentally wiping another volume

Reply:
Oh yeah, good point (!)